indico/modules/events/templates/emails/move_request_closure.txt

```diff
@@ -0,0 +1,33 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    Event move request update
+{%- endblock %}
+
+{% block header -%}{%- endblock %}
+
+{% block body -%}
+    {%- filter dedent -%}
+        {% if accept %}
+            The following events have been moved:
+        {% else %}
+            Move requests of the following events have been rejected:
+        {% endif %}
```

Comment:
Maybe we could use different wording if there's just one event... (that's going to be the most common case after all!)